scheduler/src/cook/components.clj

```diff
@@ -64,26 +65,55 @@
     (require (symbol ns))
     (resolve var-sym)))
 
+(defn tell-jetty-about-usename [h]
```

Comment:
This commit just moves the function to a different place in the same file, but I agree that `usename` probably wasn't intentional.